components/Navbar.js

```diff
@@ -0,0 +1,107 @@
+// import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+// import gmData from '../../config/gmData';
+
+//to allow GM page to be visible on the navbar and accessible
+//uncomment the above import line "import gmData from '../../config/gmData';" to get quarter from the gmData page
+//uncomment the two lines saying  "/* <Link href="/gm"><li>{gmData.date.quarter} gm</li></Link> */"
```

Comment:
As this is written right now, this is kind of a hack, but it's just a result of how we designed the `Navbar` component

I'm curious what y'alls thoughts are on making this more friendly to page inactivation. We could easily store the `ul` information in a json object and have the Navbar be constructed from it. I think this would also benefit us by making the code DRYer.

Reply:
Yeah, I'm not the largest fan of this either, especially since this requires people to update comments (uncommon) when implementation details change, like they did here.

The JSON implementation is a good idea; our friends at Creative Labs do something similar, see https://github.com/UCLA-Creative-Labs/sunshine/blob/master/components/Navbar.tsx#L9